files/en-us/web/http/authentication/index.html

```diff
@@ -34,7 +34,7 @@ <h3 id="Proxy_authentication">Proxy authentication</h3>
 
 <h3 id="Access_forbidden">Access forbidden</h3>
 
-<p>If a (proxy) server receives valid credentials that are inadequate to access a given resource, the server should respond with the {{HTTPStatus("403")}} <code>Forbidden</code> status code. Unlike {{HTTPStatus("401")}} <code>Unauthorized</code> or {{HTTPStatus("407")}} <code>Proxy Authentication Required</code>, authentication is impossible for this user.</p>
+<p>If a (proxy) server receives valid credentials that are inadequate to access a given resource, the server should respond with the {{HTTPStatus("403")}} <code>Forbidden</code> status code. Unlike {{HTTPStatus("401")}} <code>Unauthorized</code> or {{HTTPStatus("407")}} <code>Proxy Authentication Required</code>, authentication is impossible for this user.the 401 response points that the request lacks the required authentication credentials, here the user may send a new request or replace the existing Authorization header field</p>
```

Comment:
What about:

```suggestion
<p>If a (proxy) server receives <em>invalid</em> credentials, it should return a {{HTTPStatus("401")}} <code>Unauthorized</code> or a {{HTTPStatus("407")}} <code>Proxy Authentication Required</code>, and the user may send a new request or replace the {{HTTPHeader("Authorization")}} header field.</p>
<p>If a (proxy) server receives valid credentials that are <em>inadequate</em> to access a given resource, the server should respond with the {{HTTPStatus("403")}} <code>Forbidden</code> status code. Unlike {{HTTPStatus("401")}} <code>Unauthorized</code> or {{HTTPStatus("407")}} <code>Proxy Authentication Required</code>, authentication is impossible for this user and browsers will not propose a new attempt.</p>
<p>In all cases, the server may prefer returning a {{HTTPStatus("404")}} <code>Not Found</code> status code, to hide the existence of the page to user without adequate privileges or not correctly authenticated.</p>
```
I prefer this order (401 is more common than 403, and 404 needs to be mentioned).

Reply:
Thanks a lot, teoli, i will surely change the above as you said